### ISSUE TITLE
after pause video the view show thick gray line ?

### ISSUE THREAD

**Issue by Mohammed-Eliass** [gilesvangruisen/Swift-YouTube-Player#113]
after pausing the video the view of youtubePlayer showing thick gray line like this image
![screen shot 2017-03-14 at 2 58 05 pm](https://cloud.githubusercontent.com/assets/18654470/23900075/6cc33818-08c0-11e7-969d-24dbe34cca9d.png)

How to remove it ?

**Comment by UshaRao** [comment]
Am also experiencing same thing. Did You get solution for this, If yes could you please share answer here. 
Thank you

**Comment by Mohammed-Eliass** [comment]
yes,I moved the constraint to go under the navigation item. (Stupid solution), for better results I think you should change the library.

Sorry bro. :(
